spec/cc/analyzer/filesystem_spec.rb

```diff
@@ -89,5 +89,22 @@ module CC::Analyzer
         ].sort)
       end
     end
+
+    describe "#write" do
+      it "writes to the filesystem, given a path to a file and content" do
+        root = Dir.mktmpdir
+        Dir.mkdir(File.join(root, "foo"))
+        Dir.mkdir(File.join(root, "bar"))
+        File.write(File.join(root, "foo.rb"), "")
+        File.write(File.join(root, "foo", "bar.rb"), "")
+
+        filesystem = Filesystem.new(root)
+
+        filesystem.write("foo/bar.js", "Hello world")
+
+        filesystem.exist?("foo/bar.js").must_equal(true)
+        filesystem.read_path("foo/bar.js").must_equal("Hello world")
```

Comment:
Doh. I didn't realize `#read_path` existed. Please remove the `#read` method I added and use `#read_path` instead.